src/templates/edit.html

```
@@ -94,7 +94,7 @@ <h3 data-action='toggle-next' data-toggle-target-extra='mainTextEllipsis' data-o
           <textarea aria-label='{{ 'Main content'|trans }}' id='body_area' class='mceditable invisible' name='body'>{{ Entity.entityData.body }}</textarea>
         {% endif %}
       </div>
-      <div class='d-flex ml-1'>
+      <div data-scroll-btn='fa-font' data-scroll-btn-label='{{ 'Main text end'|trans }}' data-scroll-btn-y='42' class='d-flex ml-1'>
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)